library/signed/src/Tests.qs

```diff
@@ -8,47 +8,24 @@ import Measurement.MeasureSignedInteger;
 /// This entrypoint runs tests for the signed integer library.
 operation Main() : Unit {
     UnsignedOpTests();
-    MeasureSignedIntTests();
+    Fact(Qtest.Operations.CheckAllTestCases(MeasureSignedIntTests()), "SignedInt tests failed");
     SignedOpTests();
```

Comment:
Why did only this middle case get updated but not the other two around it?

Reply:
per the description, I wanted to convert at least one test case to make sure we are happy with the API and have an example of it being used. However, I think upgrading all tests is out of scope for the PR, and it would be unwise to convert everything before the API on this library is finalized (i.e. the PR is merged)